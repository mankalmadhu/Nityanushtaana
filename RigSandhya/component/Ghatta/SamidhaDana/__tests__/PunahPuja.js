import React from 'react';
import ಪುನಃಪೂಜಾ from '../PunahPuja';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../Gayatri/Trihruti', () => 'ತ್ರಿಹೃತಿ');

test("ಪುನಃಪೂಜಾ", () => {
   expect(Renderer.create(
    <ಪುನಃಪೂಜಾ />
  ).toJSON()).toMatchSnapshot();

});