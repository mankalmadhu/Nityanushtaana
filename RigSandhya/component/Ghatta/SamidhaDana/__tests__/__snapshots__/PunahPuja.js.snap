// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪುನಃಪೂಜಾ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಪುನಃಪೂಜಾ
  </ಶೀರ್ಷಿಕೆ>
  <ಶೀರ್ಷಿಕೆ>
    (ಅಗ್ನಿಗೆ ಅಭಿಮುಖವಾಗಿ ಕುಳಿತು)
  </ಶೀರ್ಷಿಕೆ>
  <ವಿನಿಯೋಗ>
    ಹೋಮಾ೦ತೇ ಆಗ್ನಿ ಪುರುಷಾಯ ನಮಃ । ಅಕ್ಷತಾ೦ ಸಮರ್ಪಯಾಮಿ । ಮ೦ಗಳದ್ರವ್ಯಾಣಿ ಸಮರ್ಪಯಾಮಿ ।
    <ತ್ರಿಹೃತಿ />
    <ತ್ರಿಹೃತಿ />
    ಯಥಾ ಭಾಗಶಃ ಆಜ್ಯೋಪಹಾರ ನಿವೇದನ೦ ಸಮರ್ಪಯಾಮಿ ।
  </ವಿನಿಯೋಗ>
</View>
`;
